mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.EraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
+      if (!deletedOrder) {
+        return res.status(404).json({ error: "No such order found" });
```

Comment:
The error message should be consistent throughout the codebase. In other parts of the code, the error response is in the format `{ error: errorHandler(err) }`. This should be followed here as well. 

Suggested improvement: `return res.status(404).json({ error: errorHandler("No such order found") });`